### PR TITLE
Feature/ui integrate spark

### PR DESCRIPTION
**Pull request by nistala**
screenshot from 2015-04-22 20 10 54
screenshot from 2015-04-22 20 10 43
screenshot from 2015-04-22 20 10 29
screenshot from 2015-04-22 20 09 50
Create pages for spark.

### CODE REVIEW

File: cdap-ui/app/features/spark/controllers/tabs/runs/tabs/status-ctrl.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+      metricPaths[metricsBasePath + '.BlockManager.memory.maxMem_MB?aggregate=true'] = 'blockMaxMemory';
+      metricPaths[metricsBasePath + '.BlockManager.memory.memUsed_MB?aggregate=true'] = 'blockUsedMemory';
+      metricPaths[metricsBasePath + '.BlockManager.disk.diskSpaceUsed_MB?aggregate=true'] = 'blockDiskSpaceUsed';
+      metricPaths[metricsBasePath + '.DAGScheduler.job.activeJobs?aggregate=true'] = 'schedulerActiveJobs';
+      metricPaths[metricsBasePath + '.DAGScheduler.job.allJobs?aggregate=true'] = 'schedulerAllJobs';
+      metricPaths[metricsBasePath + '.DAGScheduler.stage.failedStages?aggregate=true'] = 'schedulerFailedStages';
+      metricPaths[metricsBasePath + '.DAGScheduler.stage.runningStages?aggregate=true'] = 'schedulerRunningStages';
+      metricPaths[metricsBasePath + '.DAGScheduler.stage.waitingStages?aggregate=true'] = 'schedulerWaitingStages';
+      
+      angular.forEach(metricPaths, function (path, name) {
```
Comment (ajainarayanan) [inline cdap-ui/app/features/spark/controllers/tabs/runs/tabs/status-ctrl.js]: We don't seem to be using the response. Do we need to poll for so many metrics?

File: cdap-ui/server/config/development/cdap.json [hunk 2]
```diff
@@ -110,7 +110,7 @@
   "app.exec.threads": "20",
   "stream.bind.address": "127.0.0.1",
   "data.tx.snapshot.dir": "/cdap/tx.snapshot",
-  "dashboard.bind.port": "9999",
```
Comment (ajainarayanan) [inline cdap-ui/server/config/development/cdap.json]: This should not be in the PR right? We now have removed the old UI. So 9999 port should be available for us.

File: cdap-ui/app/features/spark/controllers/tabs/runs/tabs/status-ctrl.js (hunk 1 above)
Comment (nistala) [inline cdap-ui/app/features/spark/controllers/tabs/runs/tabs/status-ctrl.js]: I mentioned on hipchat that the metric paths are not right, once we figure out the paths, I will use the response. These the metrics the old UI polls for.

**Comment by anwar6953** [comment]
Empty file?
https://github.com/caskdata/cdap/pull/2236/files#diff-00d8eabafdf20b0479edf13e588c3133

**Comment by sreevatsanraman** [comment]
LGTM from @ajainarayanan he fixed some comments that needs to be fixed.

**Pull request merged**